README.md

```
@@ -1,40 +1,90 @@
-[中文](README.zh.md) | English
+# wuji-retargeting
 
-# WujiHand Retargeting
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE) [![Release](https://img.shields.io/github/v/release/wuji-technology/wuji-retargeting)](https://github.com/wuji-technology/wuji-retargeting/releases)
 
-Hand pose retargeting system for WujiHand using Wuji Retargeting algorithm.
+Hand pose retargeting system for Wuji Hand. High-precision retargeting based on adaptive analytical optimization, supporting Apple Vision Pro hand tracking for real-time teleoperation.
 
-## Apple Vision Pro Setup
+https://github.com/user-attachments/assets/72116289-7a33-4a6b-83ca-fb4d9aaece0d
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)